.gitattributes

```diff
@@ -0,0 +1,4 @@
+# iOS localizable strings
```

Comment:
I think we should avoid this if we can.

Can we instead just convert the files (by hand at first) to utf-8 before adding them to the repo?

Then in the future, we can add that into the download script if necessary.